resources/public/index.ejs

```diff
@@ -79,6 +79,18 @@
         window.clientConfig = {};
       }
     </script>
+    <script>
+      const regex = /\?applicationId=(\d+)/;
```

Comment:
Actually this requires that the `applicationId` is the first query parameter. We should probably change it to become more flexible with `/\applicationId=(\d+)/` (untested!).

Reply:
Thank you! I have tested it and it works as expected